src/components/ContextEdit/index.tsx

```diff
@@ -190,20 +200,21 @@ const ContextEdit: FC<Props> = ({ show, onHide, variant, data, id }) => {
                   )}
                 </FieldArray>
                 <Divider />
-                <div className={styles.row}>
-                  <Button variant="negative" onClick={onHide}>
-                    CANCEL
-                  </Button>
-                  <div className={'flex-grow-1'} />
-                  <Button type={'submit'} variant="primary" onClick={() => handleSubmit()}>
-                    EDIT
-                  </Button>
-                </div>
+                <div className={styles.row}></div>
               </>
             );
           }}
         </Formik>
       </Modal.Body>
+      <Modal.Footer>
+        <Button variant="negative" onClick={onHide}>
+          CANCEL
+        </Button>
+        <div className={'flex-grow-1'} />
+        <Button type={'submit'} variant="primary" onClick={() => submitWired()}>
+          SAVE
+        </Button>
+      </Modal.Footer>
```

Comment:
Can you move both buttons to right. Having them on the both ends of the form is quite unusual.